### PR TITLE
Adding link to bundling docs

### PR DESCRIPTION
**Pull request by JonathanCrd**
Solves https://github.com/Azure/autorest.typescript/issues/1214 for v4.

Bullet point referencing the Javascript Bundle added under the `browser - Authentication, client creation, and ...` section.

### CODE REVIEW

File: src/vanilla/Model/CodeModelTS.cs [hunk 1]
```diff
@@ -1098,6 +1098,7 @@ private void GenerateHowToUseInBrowser(MarkdownBuilder builder)
                 builder.Line("In browser applications, we recommend using the `InteractiveBrowserCredential` that interactively authenticates using the default system browser.");
                 builder.Line("  - See [Single-page application: App registration guide](https://docs.microsoft.com/azure/active-directory/develop/scenario-spa-app-registration) to configure your app registration for the browser.");
                 builder.Line("  - Note down the client Id from the previous step and use it in the browser sample below.");
+                builder.Line("  - To use this client library in the browser, first you need to use a bundler. For details on how to do this, please refer to our bundling documentation.");
```
Comment (joheredi) [inline src/vanilla/Model/CodeModelTS.cs:1101]: Can you add a link to the bundling docs?

Comment (JonathanCrd) [reply src/vanilla/Model/CodeModelTS.cs:1101]: My bad, Thanks for spotting this!

**Review by joheredi — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
Don't we have to update the package version or something to ensure a new version is released with this change?

**Comment by joheredi** [comment]
Yes, we need to bump the patch version before releasing it. I usually run `npm version patch` and push the changes. @JonathanCrd ping me when this is ready to release and I can help with that

**Comment by sarangan12** [comment]
@joheredi The CI is not running for this PR. Any ideas??

**Comment by JonathanCrd** [comment]
This is the output of the pipelines: 
```
##[warning]An image label with the label Ubuntu 16.04 does not exist.
,##[error]The remote provider was unable to process the request.
```

Could the service or VMs be down?

**Comment by joheredi** [comment]
It looks like the Ubuntu 16 vm image has been discontinued in Azure DevOps. You should be able to fix this updating `azure-pipelines.yml`.

This change should fix it:
image

**Comment by ramya-rao-a** [comment]
By the end of January 2022, almost all mgmt plane packages would have moved to using v6 code generator. Therefore, it is not worth our time to fix this for the v4 code generator

**Pull request closed without merging**